### PR TITLE
Enhancement/9643 one tap copy

### PR DESCRIPTION
**Pull request by zutigrm**
## Summary

Addresses issue:

- #9643

## Relevant technical choices

There were a lot of details on the edit form that do not match with the Figma design, so I polished it here, together with learn more change, to avoid delaying SiWG epic, since we have bug bash coming in few days

## PR Author Checklist

- [x] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 5.2 and PHP 7.4.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [ ] Run the code.
- [ ] Ensure the acceptance criteria are satisfied.
- [ ] Reassess the implementation with the IB.
- [ ] Ensure no unrelated changes are included.
- [ ] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [ ] Ensure there is a QA Brief.
- [ ] Ensure there are no unexpected significant changes to file sizes.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

### CODE REVIEW

**Comment by zutigrm** [comment]
Note: 3 KMW VRT updated are not related to this change, but seems to be originating from develop merged previously from other PR, I fixed it here

File: assets/js/modules/sign-in-with-google/components/settings/SettingsForm.js [hunk 1]
```diff
@@ -30,46 +30,95 @@ import {
...
-					</Cell>
-				</Row>
-			</Grid>
+						</Cell>
+						<Cell size={ 12 }>
+							<OneTapToggle />
+						</Cell>
+					</Row>
+				</Grid>
+			) }
```
Comment (tofumatt) [inline assets/js/modules/sign-in-with-google/components/settings/SettingsForm.js]: Let's not use ternaries for big components like this: they get quite hard-to-follow/place.

I'll make the change here, but just a note going forward 🙂

Comment (tofumatt) [reply assets/js/modules/sign-in-with-google/components/settings/SettingsForm.js]: Also though: this seems like something we can handle with CSS, this seems a bit complicated/has a lot of duplication. Let's find a CSS-only approach. 🤔

Comment (tofumatt) [reply assets/js/modules/sign-in-with-google/components/settings/SettingsForm.js]: What was the reason for this component split based on viewport size? I removed it and it worked just fine with the "large" viewport code.

**Review by tofumatt — DISMISSED** [review]
I've removed the `isSmallScreen` code because it didn't seem necessary, but I'll make the changes and confirm it doesn't break VRTs.

If so I'll get this merged with my changes.

**Review by tofumatt — APPROVED** [review]

File: assets/js/modules/sign-in-with-google/components/settings/SettingsForm.js (hunk 1 above)
Comment (zutigrm) [reply assets/js/modules/sign-in-with-google/components/settings/SettingsForm.js]: The order of the elements within the grid, in new layout is showing well for desktop, but layout change was needed to re-order one tap toggle on mobile and tablet, so they are above the preview box

**Pull request merged**